code/modules/client/client_procs.dm

```diff
@@ -257,6 +257,10 @@
 	//CONNECT//
 	///////////
 /client/New(TopicData)
+	// TODO: Remove with 516
+	if(byond_version >= 516) // Enable 516 compat browser storage mechanisms
+		winset(src, "", "browser-options=byondstorage")
```

Comment:
How long are we gonna support old clients for? Not sure if `TODO` is applicable here.

Reply:
I'd prefer we drop them ASAP so TGUI doesn't have one foot in the grave with IE11 support

Reply:
Valid